webdriver-spec.html

```diff
@@ -2882,6 +2888,68 @@
  whose <a><code>readOnly</code> attribute</a> is set.
 
 <section>
+<h3>Element Interactibility</h3>
+
+<p>In order to determine if an element can be interacted with,
+ WebDriver performs hit-testing to find
+ if the interaction will be able to reach the requested element.
+
+<p>An <dfn>interactible element</dfn> is defined to be
+ the first non-transparent <a>element</a>,
+ defined by the paint order found at the centre point
+ of its rectangle that is inside the <a>viewport</a>,
+ excluding the size of any rendered scrollbars.
+
+<p>In order to produce an <dfn>interactible elements tree</dfn>
+ from an <var><a>element</a></var>:
+
+<ol>
+ <li><p>If <var>element</var> is <a>not in the same tree</a>
+  as the <a>current browsing context</a>’s <a>active document</a>,
+  return an empty sequence.
+
+ <!--
+  no need to care for scroll bars here
+  because elementsFromPoint takes care of it
+
+  this is the same as
+
+    rect.width - (rect.right - innerWidth)
+    rect.height - (rect.bottom - innerHeight)
+ -->
+ <li><p>Let <var>visible width</var> be
+  the <var>element</var>’s <a>bounding rectangle</a>’s
+  (<a>max</a>(<a>x coordinate</a>, <a>x coordinate</a> + <a>width dimension</a>) –
```

Comment:
See the definition under _Terminology_.  It’s a way to get the largest value of a variable list of values.